docs/source/modin/supported/series_supported.rst

```diff
@@ -221,10 +223,6 @@ Methods
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
 | ``hist``                    | N                               |                                  |                                                    |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
-| ``idxmax``                  | N                               |                                  |                                                    |
```

Comment:
I think `idxmax` and `idxmin` should also work for Series. We also have tests for it.
https://pandas.pydata.org/docs/reference/api/pandas.Series.idxmax.html

Reply:
They are, these were just duplicated. I just realized they're not alphabetized properly though, so I've moved `hist` to the proper location.